apps/pom.xml

```diff
@@ -187,15 +189,15 @@
 					Framework that we're using, per http://docs.spring.io/spring-data/jpa/docs/1.11.4.RELEASE/reference/html/#dependencies.spring-framework. -->
 				<groupId>org.springframework.data</groupId>
 				<artifactId>spring-data-jpa</artifactId>
-				<version>1.11.23.RELEASE</version>
+				<version>2.7.1</version>
```

Comment:
can you confirm that this version is compatible with the new version of Spring and update the comment above accordingly?

Reply:
lets add this version to the properties above

Reply:
Updated